docs/UserGuide.md

```diff
@@ -183,6 +189,8 @@ Format: `deletetag [-force] t/TAG1…​`
 * Tag names are **case-insensitive**. e.g. `deletetag t/BRIDE'S SIDE` is the same as `deletetag t/Bride's Side`.
 * Additional leading entries will be ignored. e.g. `deletetag 1 t/bride's side` is the same as `deletetag t/bride's side`.
 * If any guest currently has the tag `friends`, `deletetag t/friends` will not work. To force delete the tag and remove it from all guests, use `deletetag -force t/friends`.
+* Tag names are **case-insensitive**. e.g `deletetag t/BRIDE'S SIDE` is the same as `deletetag t/Bride's Side`.
+* Attempting to delete a mix of tags with invalid and valid names at once will result in an error.
```

Comment:
Wording is inconsistent with the corresponding line in the newtag section (tags vs mix of tags)